integration/registry_docker_config.go

```diff
@@ -0,0 +1,104 @@
+package integration
```

Comment:
This file should get renamed from `integration/registry_docker_config.go`  to `integration/registry_docker_config_test.go`. That will make the GuardRails scan happy too.